lib/elixir/test/erlang/string_test.erl

```diff
@@ -22,7 +22,7 @@ extract_interpolations_without_interpolation_test() ->
 
 extract_interpolations_with_escaped_interpolation_test() ->
   ["f\\#{o}o"] = extract_interpolations("f\\#{o}o"),
-  {1, 8, ["f\\#{o}o"], [], _} =
```

Comment:
I believe this test was asserting the incorrect thing. If I'm reading it correctly, the setup here is that we're tokenizing the following string and we expect this number to be the column after interpolations are extracted:

```elixir
"f\#{o}o"
 ^       ^
 2       10
```